recipes/mayavi/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set version = "4.5.0" %}
+
+package:
+  name: mayavi
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/enthought/mayavi.git
+  git_rev: {{ version }}
+
+build:
+  entry_points:
+    - mayavi2 = mayavi.scripts.mayavi2:main
+    - tvtk_doc = tvtk.tools.tvtk_doc:main
+  skip: True  # [osx or (linux and py34) or win]
+  number: 0
+
+requirements:
+  build:
+    - toolchain
+    - python
+    - numpy
```

Comment:
The [doc](http://docs.enthought.com/mayavi/tvtk/README.html#installation) says: "TVTK is meant to be installed as part of the mayavi package.", so it must be ok to have 1 package